cmake_modules/dependencies/el5.cmake

```diff
@@ -1 +1 @@
-SET_DEPENDENCIES ( CPACK_RPM_PACKAGE_REQUIRES boost141-regex openldap libicu m4 libtool libxslt libxml2 gcc-c++ openssh-server openssh-clients expect libarchive rsync apr apr-util zip tbb )
+SET_DEPENDENCIES ( CPACK_RPM_PACKAGE_REQUIRES boost141-regex openldap libicu m4 libtool libxslt libxml2 gcc-c++ openssh-server openssh-clients expect libarchive rsync apr apr-util zip tbb numactl-devel )
```

Comment:
I couldn't find a non development package that included the library - if there is one then please let me know what it is.